app/frontend/src/components/forms/RequestReceipt.vue

```diff
@@ -66,7 +66,7 @@ export default {
             to: this.to,
           });
           this.addNotification({
-            type: NotificationTypes.SUCCESS,
+            ...NotificationTypes.SUCCESS,
```

Comment:
Generally better JS convention to list attributes coming in via spread operator last - I'd recommend making message come first, then this line.

Reply:
okay will change in my next commit